diff --git a/cache-dir/head/utils.js b/cache-dir/head/utils.js
index f3f48087fd9cf448c21aa2621e87e4acb2554372..6cefc203f6a7042be5be054b0f2716a5b1d45572 100644
--- a/cache-dir/head/utils.js
+++ b/cache-dir/head/utils.js
@@ -153,13 +153,8 @@ export function getValidHeadNodesAndAttributes(
           }
         }
       } else {
-        let clonedNode = node.cloneNode(true)
+        let clonedNode = massageScript(node)
         clonedNode.setAttribute(`data-gatsby-head`, true)
-
-        // // This is hack to make script tags work
-        if (clonedNode.nodeName.toLowerCase() === `script`) {
-          clonedNode = massageScript(clonedNode)
-        }
         // Duplicate ids are not allowed in the head, so we need to dedupe them
         if (id) {
           if (!seenIds.has(id)) {
@@ -196,7 +191,7 @@ export function getValidHeadNodesAndAttributes(
 }
 
 function massageScript(node) {
-  const script = document.createElement(`script`)
+  const script = document.createElement(node.nodeName.toLowerCase())
   for (const attr of node.attributes) {
     script.setAttribute(attr.name, attr.value)
   }
